partner_communication/models/communication_config.py

```diff
@@ -179,6 +181,19 @@ def _validate_attachment_function(self):
     ##########################################################################
     #                             PUBLIC METHODS                             #
     ##########################################################################
+    @api.model
+    def get_config_for_lang(self, lang):
+        specific_config_found = False
+        for config in self.omr_config_ids:
+            if config.lang_id:
+                if config.lang_id == lang:
```

Comment:
This test is sufficient. If config.lang_id is not set, it won't equal to lang...
```
if config.lang_id == lang:
```